sdk/go/common/workspace/templates.go

```diff
@@ -51,6 +48,17 @@ const (
 	pulumiLocalPolicyTemplatePathEnvVar = "PULUMI_POLICY_TEMPLATE_PATH"
 )
 
+var (
```

Comment:
Would it be worth adding a comment explaining _why_ these are variables rather than constants so we don't unknowingly revert in the future?

Reply:
Yes, that's probably a good idea.

Reply:
This is addressed in `5b71f81`.